### ISSUE TITLE
The deprecation warning of 'fs.close(fd, callback)'

### ISSUE THREAD

**Issue by renhongxu123** [alanshaw/markdown-pdf#119]
Since node ### v7.0.0, the callback parameter is ### no longer optional. Not passing it will emit a deprecation warning.
https://nodejs.org/dist/latest-v7.x/docs/api/fs.html#fs_fs_close_fd_callback.

Please fix it, thx a lot.

**Comment by anko** [comment]
Closing as duplicate of https://github.com/alanshaw/markdown-pdf/issues/114.

PR https://github.com/alanshaw/markdown-pdf/pull/118 fixes those warnings—it's waiting on @alanshaw to merge, as he holds the npm-publish rights. In the meantime, the warnings are safe to ignore.
